src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+interface TodoInfoProps {
+  todo: Todo
+}
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+export const TodoInfo = ({ todo }: TodoInfoProps) => {
+  const getUserById = (id: number): User => {
```

Comment:
The `getUserById` function should be moved outside of the `TodoInfo` component to prevent it from being recreated on every render.